alws/crud/build_node.py

```diff
@@ -48,53 +48,50 @@ async def get_available_build_task(
     db: AsyncSession,
     request: build_node_schema.RequestTask,
 ) -> typing.Optional[models.BuildTask]:
-    async with db.begin():
-        # TODO: here should be config value
-        ts_expired = datetime.datetime.utcnow() - datetime.timedelta(
-            minutes=20
-        )
-        db_task = await db.execute(
-            select(models.BuildTask)
-            .where(~models.BuildTask.dependencies.any())
-            .with_for_update()
-            .filter(
-                sqlalchemy.and_(
-                    models.BuildTask.status < BuildTaskStatus.COMPLETED,
-                    models.BuildTask.arch.in_(request.supported_arches),
-                    sqlalchemy.or_(
-                        models.BuildTask.ts < ts_expired,
-                        models.BuildTask.ts.is_(None),
-                    ),
-                )
-            )
-            .options(
-                selectinload(models.BuildTask.ref),
-                selectinload(models.BuildTask.build).selectinload(
-                    models.Build.repos
-                ),
-                selectinload(models.BuildTask.platform).selectinload(
-                    models.Platform.repos
-                ),
-                selectinload(models.BuildTask.build).selectinload(
-                    models.Build.owner
+    # TODO: here should be config value
```

Comment:
It was a pre-existing TODO even before my changes, so I'm not really aware of what exactly was meant by this. As far as i can see from git blame in master, it seems that this line was commited 3 years ago by Vasiliy 
I can create the issue but won't be able to provide an extensive description, is it alright?

Reply:
Sure, I guess that the task is about not hardcoding the time to consider tasks expired (20 mins) and make it configurable. This would involve changes in both web-server and albs-deploy

Reply:
Done! https://github.com/AlmaLinux/build-system/issues/284